etc/testing/travis_install.sh

```diff
@@ -1,88 +0,0 @@
-#!/bin/bash
-
-set -ex
-
-# install latest version of docker
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-sudo apt-get update -y
-sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
-
-# install goreleaser
-pushd /usr/local/
-curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sudo sh
-popd
-goreleaser -v
-
-# reconfigure & restart docker
-echo 'DOCKER_OPTS="-H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null
-echo '{"experimental":true}' | sudo tee /etc/docker/daemon.json
-sudo service docker restart
-
-# Install deps
-sudo apt-get install -y -qq \
-  jq \
-  silversearcher-ag \
-  python3 \
-  python3-pip \
-  python3-setuptools \
-  pkg-config \
-  fuse
-
-# Install fuse
-sudo modprobe fuse
```

Comment:
@lukemarsden Likewise how does fuse get installed in testfaster?

Reply:
oh, I don't know if it does. what would be failing if it weren't?